users/details/script.js

```diff
@@ -350,6 +371,28 @@ function generateReadableDateFromSecondsTimeStamp(timeStamp) {
   //created function for readable date format
   return new Date(timeStamp * 1000).toDateString(); // new function because we are getting the value in seconds and not milliseconds
 }
+console.log(generateReadableDateFromSecondsTimeStamp('1688445662'));
+
+function generateDaysToGo(dateStr, status) {
+  const inputDate = new Date(dateStr);
+
+  const now = new Date();
+  const offset = 330 * 60 * 1000;
+  const currentDate = new Date(now.getTime() + offset);
+  const diff = inputDate - currentDate; // Calculates the difference in milliseconds
+  if (diff <= 0 && status == 'COMPLETED') {
+    return 'Task Completed Within Deadline';
```

Comment:
what about below message. 

"Task has been completed within Committed timeline."

Reply:
successfully made the changes.
![image](https://github.com/Real-Dev-Squad/website-dashboard/assets/31709147/6475f40d-c1a3-4eb1-adf3-0a8ff56a23ef)